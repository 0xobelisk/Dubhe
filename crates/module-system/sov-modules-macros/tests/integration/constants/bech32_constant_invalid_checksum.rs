use sov_bank::TokenId;
use sov_modules_api::macros::config_bech32;

const TEST_TOKEN_ID_INVALID_CHECKSUM: TokenId =
    config_bech32!("TEST_TOKEN_ID_INVALID_CHECKSUM", TokenId);

fn main() {
    assert_eq!(TEST_TOKEN_ID_INVALID_CHECKSUM, TokenId::from([0; 32]));
}
