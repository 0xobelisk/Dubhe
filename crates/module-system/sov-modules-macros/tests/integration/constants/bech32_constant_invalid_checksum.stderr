error: The constant named `TEST_TOKEN_ID_INVALID_CHECKSUM` inside `$WORKSPACE/constants.testing.toml` is not a valid bech32 string; decoding failed: parse failed
 --> tests/integration/constants/bech32_constant_invalid_checksum.rs:5:20
  |
5 |     config_bech32!("TEST_TOKEN_ID_INVALID_CHECKSUM", TokenId);
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
